tests/functional/ui/test_error_improvements.py

```
@@ -18,12 +22,65 @@
+    """
+    namespace = config.ENV_DATA.get("cluster_namespace", "openshift-storage")
+
+    if pool_type == "block":
+        try:
+            OCP().exec_oc_cmd(
+                f"delete cephblockpool {pool_name} -n {namespace} "
+                "--force --grace-period=0"
+            )
+            logger.info(f"Force-deleted CephBlockPool {pool_name}")
+        except CommandFailed:
+            logger.debug(f"CephBlockPool {pool_name} already deleted or not found")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)